fetch/rest.ts

```diff
@@ -0,0 +1,6 @@
+const request = require('request');
+export const req = request('http://www.google.com', function (error:any, response:any, body:any) {
+  console.log('error:', error); // Print the error if one occurred
+  console.log('statusCode:', response && response.statusCode); // Print the response status code if a response was received
+  console.log('body:', body); // Print the HTML for the Google homepage.
```

Comment:
@dks-ar lgtm, remove this line, body could be huge.

Reply:
@dhieru thanks for the review, fixed in `318d5780`